src/common.rs

```diff
@@ -992,6 +1062,18 @@ mod tests {
         println!("{} {:?}", ProcessStatus::Parked, ProcessStatus::Idle);
     }
 
+    #[test]
+    fn from_str_mac_address() {
+        let mac = MacAddr::from_str("e5:5d:59:e9:6e:b5").unwrap();
```

Comment:
Please also add some failing cases to check that they don't work.

Reply:
A testcase named `test_mac_address_from_str_fail` is added👌